config/config/sc-config.yaml

```diff
@@ -301,7 +301,17 @@ dex:
     requests:
       cpu: 5m
       memory: 25Mi
-  tolerations: []
+  tolerations:
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/master
+      operator: Equal
```

Comment:
"kubernetes.io/os" is better maybe so it covers all nodes?

Reply:
No, I meant change back to `tolerations: []` so that it does not run on control-plane nodes.
Or why did you want them to run on the control-plane nodes?

Reply:
Hmm, interesting. I sat "tolerations" to make "topologySpreadConstraints:" work but now it doesn't seem to be needed :+1: